Magento_Sales/templates/order/totals.phtml

```diff
@@ -4,19 +4,19 @@
         <?= $block->getChildHtml($_total->getBlockName(), false) ?>
     <?php else: ?>
         <tr class="<?= $block->escapeHtmlAttr($_code) ?>">
-            <td <?= $block->getLabelProperties() ?> colspan="4">
+            <td <?= $block->escapeHtmlAttr($block->getLabelProperties()) ?> colspan="4">
```

Comment:
We can't use this for arguments that have string like `someAttribute="someValue"`. We use it just for "value" part of attributes. So here we should have noEscape like in original file and escpaing should/is done inside getLabelProperties() method